src/odemis/acq/stream/_projection.py

```diff
@@ -1131,6 +1167,7 @@ class NeedRecomputeException(Exception):
                 # image changed
                 need_recompute = True
 
+        logging.debug(f"{self.stream.name.value} (rect size: {rect_x*1e3:.4f} x {rect_y*1e3:.4f} mm) read {x2-x1} x {y2-y1} tiles at zoom: {z}, mpp: {self.mpp.value} m/px")
```

Comment:
Probably passes our CI linter, as we made it super lax (ie, it only cares about spaces). Moreover, there is no "official" rule in the coding guidelines (probably just because we forgot to define it). Still, typically you want a line < 100 char, or at most < 120 chars. 
@patrickcleeve2, could you break this line, or maybe just drop it if you don't think this log is useful now that the bug is fixed.

Reply:
I've split the line, i'd like to leave the debug info in the log as it makes it easier to investigate any of these issues if they happen for customers.